lambda_function.py

```
@@ -3,223 +3,290 @@
 s3 = boto3.client('s3')
 
-def read_trajectory_from_s3(bucket, key):
-    """Read SLAM trajectory file from S3"""
+
+# --------------------------------------------------------------------------- #
+# Data loading utilities
+# --------------------------------------------------------------------------- #
+def read_trajectory_from_s3(bucket: str, key: str):
+    """
+    Read a Metashape trajectory file that contains position, orientation, and
+    optionally the camera-to-world rotation matrix (r11–r33).
```

Ruff (0.14.4)

Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)